src/components/empty_state/mod.rs

```diff
@@ -0,0 +1,17 @@
+pub mod empty_state;
```

Comment:
I am not a fan of prefixing sub-modules with the name of the parent module. This ends up in a qualified name of `crate::components::empty_state::empty_state_actions`. I think they should simple be named `crate::components::empty_state::actions` (so just `actions`). Same for the others.